training/modules/hf_gptneox_modules.py

```diff
@@ -14,27 +14,73 @@
 from transformers.models.gpt_neox.modeling_gpt_neox import GPTNeoXLayer as _GPTNeoXBlock
 from transformers.models.gpt_neox.modeling_gpt_neox import GPTNeoXModel as _GPTNeoXModel
 from transformers.models.gpt_neox.configuration_gpt_neox import GPTNeoXConfig as GPTConfig
+from transformers.models.gpt_neox.modeling_gpt_neox import RotaryEmbedding
+
+
+try:
+    from flash_attn.flash_attention import FlashAttention
+    flash_attn_installed = True
+    print('>>>>> flash attention')
```

Comment:
```suggestion
    print('>>>>> using flash attention')
```

or

```suggestion
    print('>>>>> flash attention enabled')
```

Even better would be if you used Python logging instead of prints, but that's for another day.